pkg/tecdsa/dkg/preparams_storage.go

```diff
@@ -0,0 +1,158 @@
+package dkg
+
+import (
+	"encoding/hex"
+	"fmt"
+	"sort"
+	"sync"
+
+	"github.com/ethereum/go-ethereum/crypto"
```

Comment:
We don't need to hash with keccak256 specifically so I think we can get rid of the `go-ethereum` dependency and use `crypto/sha256` from stdlib. We already use it in our codebase: https://github.com/keep-network/keep-core/blob/e302e7152efd418b90f45cf0977f16fa08b5d7d9/pkg/tbtc/dkg.go#L52

Reply:
dd9d2233192032cfae5cdc76c2d8375895aef3d4